cluster/store/store.go

```diff
@@ -48,7 +48,7 @@ const (
 	// This is used to reduce disk I/O for the recently committed entries.
 	logCacheCapacity = 512
 
-	nRetainedSnapShots = 1
+	nRetainedSnapShots = 3
```

Comment:
thoughts(non-blocking): Should we make this parameter a config element that we can change ?

Reply:
Sure